cmd/immuadmin/commands/service.go

```diff
@@ -116,62 +97,70 @@ sudo ./immuadmin service immudb uninstall
 
 			var msg string
 			var localFile string
+			var execPath string
 
-			if stringInSlice("--remove-files", os.Args) || args[1] == "install" {
+			if args[1] == "install" {
 				if localFile, err = cmd.Flags().GetString("local-file"); err != nil {
 					return err
 				}
-				if localFile != "" {
-					_, err = os.Stat(localFile)
-					if os.IsNotExist(err) {
-						return errExecNotFound
-					}
+				if localFile, err = service.GetExecutable(localFile, args[0]); err != nil {
+					return err
 				}
+			}
 
-				if localFile == "" {
-					localFile = args[0]
-					if runtime.GOOS == "windows" {
-						localFile = localFile + ".exe"
-					}
-					_, err = os.Stat(localFile)
-					if os.IsNotExist(err) {
-						return fmt.Errorf("%s executable file was not found on current folder", args[0])
-					}
+			execPath = service.GetDefaultExecPath(localFile)
+
+			if stringInSlice("--remove-files", os.Args) {
+				if localFile, err = cmd.Flags().GetString("local-file"); err != nil {
+					return err
 				}
+			}
 
-				if localFile, err = filepath.Abs(localFile); err != nil {
+			if args[1] == "install" {
+				if execPath, err = service.CopyExecInOsDefault(localFile); err != nil {
 					return err
 				}
 			}
 
-			remove, _ := cmd.Flags().GetBool("remove-files")
-			if remove {
-				if err = os.Remove(localFile); err != nil {
+			// todo remove all involved files
```

Comment:
@mmeloni Is this TODO still valid?

Reply:
Yes, it make no sense this flag in my opinion, uninstall already do everything